src/components/layout/layout-v2.tsx

```diff
@@ -23,6 +24,11 @@ const FullHeightContainer = styled.div`
   flex-direction: column;
 
   min-height: 100vh;
+
+  padding-top: ${HEADER_HEIGHT_VALUE}px;
+  ${up('md')} {
+    padding-top: ${HEADER_HEIGHT_VALUE}px;
+  }
```

Comment:
I have the idea that the layout MUST work without any container. For this to work we have to provide counter paddings for the fixed navigation/header. They both belong together so we can assume they will always be on a page. Without the padding I'm not able to use the layout with some container with is a big red flag for me. Hence the idea of making the layout independent with this. This brings other problems: The container we use to wrap the content & sections will always apply the top spacing and add up. I didn't want to rely on collapsing margin's (too hard to follow and some don't even know about that). That's why I introduced the `first-child` statement to make the container behave differently when it's the very first element in the layout (assuming it's always placed in the layout directly). See the [container component](https://github.com/satellytes/satellytes.com/pull/272/files#diff-74b31a29bc40787564bd5df1ddd53394b1830233566d0a4790158fa57c36bafd) for the first-child thing.